File: WordPress/Classes/ViewRelated/Pages/PageListSectionHeaderView.m
#import "PageListSectionHeaderView.h"
#import "WPStyleGuide+Pages.h"
#import <WordPressShared/WPDeviceIdentification.h>

@interface PageListSectionHeaderView()

@property (nonatomic, strong) IBOutlet UILabel *titleLabel;

@end

@implementation PageListSectionHeaderView

- (void)setTitle:(NSString *)title
{
    self.titleLabel.text = [title uppercaseStringWithLocale:[NSLocale currentLocale]];
}

@end

File: WordPress/Classes/ViewRelated/Pages/PageListSectionHeaderView.swift
import UIKit

class PageListSectionHeaderView: UIView {

    @IBOutlet weak var titleLabel: UILabel!
    @IBOutlet weak var separator: UIView!

    func setTitle(_ title: String) {
        titleLabel.text = title.uppercased(with: .current)
    }

    override func awakeFromNib() {
        super.awakeFromNib()

        backgroundColor = .listBackground
        titleLabel.backgroundColor = .listBackground
        WPStyleGuide.applyBorderStyle(separator)
    }
}
